kong/plugins/ai-proxy/handler.lua

```diff
@@ -7,7 +7,7 @@ local AIPlugin = ai_plugin_base.define(NAME, PRIORITY)
 
 local SHARED_FILTERS = {
   "parse-request", "normalize-request", "enable-buffering",
-  "parse-sse-chunk", "normalize-sse-chunk",
+  "parse-sse-chunk", "normalize-sse-chunk", "normalize-response-header",
```

Comment:
If i understand correctly the previous behaviour was
```suggestion
  "normalize-response-header", "parse-sse-chunk", "normalize-sse-chunk",
```